### PR TITLE
Enforce osusergo build tag for releases (backport #6862)

### PR DESCRIPTION
**Pull request by sapk**
Backport #6862

### CODE REVIEW

**Review by adelowo — APPROVED** [review]

**Comment by sapk** [comment]
I don't think that related but CI seems broken on master since #6862. Maybe we should check before merging this just to be sure. Gentle ping to @go-gitea/maintainers if you can relaunch some build on master (or merge another PR).

**Comment by techknowlogick** [comment]
Thanks for ping :) seems unrelated, but I think your suggestion of merging a new PR to master is good idea (can't restart build due to updated translation file).

**Review by zeripath — APPROVED** [review]

**Pull request merged**
